src/main/webapp/app/exercises/programming/assess/repo-export/programming-assessment-repo-export-dialog.component.ts

```diff
@@ -82,7 +82,7 @@ export class ProgrammingAssessmentRepoExportDialogComponent implements OnInit {
         }
         this.exportInProgress = true;
         // The participation ids take priority over the participant identifiers (student login or team names).
-        if (this.participationIdList) {
+        if (this.participationIdList.length) {
```

Comment:
Are you sure `participationIdList` cannot be undefined?

Reply:
```suggestion
        if (this.participationIdList?.length) {
```

Reply:
might be saver to use the question mark operator here

Reply:
oh no I am not. In the client test I couldn't assign undefined. But I will fix this.

Reply:
fixed in 6908cd85d14c0b02c33ca323da1fc2f7493e7862